paths/keys/update.yaml

```diff
@@ -120,4 +121,10 @@ requestBody:
             description: NSStringLocalizedFormatKey attribute. Used in .stringsdict format.
             type: string
             example:
+          custom_metadata:
+            description: Custom metadata property name and value pairs to be associated with key.
```

Comment:
I believe it should be mentioned what happens with existing CM values on this key, because I have no idea. :)

Reply:
@jablan is it more descriptive now? :)